kolibri/plugins/learn/assets/src/views/TopicsPage/index.vue

```diff
@@ -124,7 +125,7 @@
                 :allowDownloads="allowDownloads"
                 data-test="search-results"
                 :contents="resourcesDisplayed"
-                :numCols="numCols"
+                :gridType="gridType"
```

Comment:
As someone coming to look at this, I do find the numCols calculations significantly easier to grok in terms of intention than the rather abstract 'gridType' - I do hope we can get rid of this in future.

Reply:
This wasn't actually an intentional doing away of `numCols` - the code there wasn't doing anything. I would be very happy to refactor it back to use `numCols` as it had been previously.

Reply:
Yes - I could see that it was a no-op, as the prop didn't exist on the component. This was more of an expression of an aspiration that we use a 'desired number of columns' in future grid work!